idd/Energy+.idd.in

```diff
@@ -24823,6 +24823,55 @@ AirflowNetwork:SimulationControl,
       \key Yes
       \key No
       \default No
+ A10, \field Duct Sizing Method
+      \type choice
+      \key None
+      \key MaximumVelocity
+      \key PressureLoss
+      \key PressureLossWithMaximumVelocity
+      \default None
+ N7 , \field Duct Sizing Factor
+      \type real
+      \minimum> 0.0
+      \default 1.0
+ N8 , \field Maximum Airflow Velocity
+      \type real
+      \units m/s
+      \minimum >0.0
+      \maximum 25.0
+      \default 5.0
+      \note Used only if Duct Sizing Type = MaximumVelocity or PressureLossWithMaximumVelocity.
+      \note When MaximumVelocity is entered, duct diameter is calculated at D = flow rate / 
+      \note cross section area.
+      \note When PressureLossWithMaximumVelocity is entered, duct diameter is calculated based on 
+      \note PressureLoss. The value is used to check to ensure the final velocity is less than
+      \note the maximum value. If greater, final value will be obtained from MaximumVelocity.
+      \note This field is apply for truck size, while branch size is based on total pressure drop.
+ N9 , \field Total Pressure Loss Across Supply Truck
+      \type real
+      \units Pa
+      \minimum >0.0
+      \note Used only if Duct Sizing Type = PressureLoss or PressureLossWithMaximumVelocity.
+      \note When PressureLoss is entered, duct diameter is calculated using Colebrook's equation  
+      \note When PressureLossWithMaximumVelocity is entered, duct diameter is calculated based on 
+      \note PressureLoss. The value is used to check to ensure the final velocity is less than
+      \note the maximum value. If greater, final value will be obtained from MaximumVelocity.
+      \note This field is apply for truck size, while branch size is based on total pressure drop.
+ N10, \field Total Pressure Loss Across Supply Branch
+      \type real
+      \units Pa
+      \minimum >0.0
+      \note Duct diameter is calculated using Colebrrook's equation  
+ N11, \field Total Pressure Loss Across Return Trunk
+      \type real
+      \units Pa
+      \minimum >0.0
+      \note Duct diameter is calculated using Colebrrook's equation  
```

Comment:
Should the notes here be the same as for "Total Pressure Loss Across Supply Trunk"?

Reply:
@mjwitte The maximum velocity is use for supply trunk. The rest of sections do not use Maximum velocity. However, when there is no solution obtained, velocity = 5 m/s is used to calculate duct diameter, so that simulation continues and warnings are issued.

The notes are revised in the last 3 fields as below.

      \note Duct diameter is calculated using Colebrook's equation. When there is no solution, 
      \note velocity = 5 m/s is used to calculate duct diameter.